tfhe/src/core_crypto/algorithms/test/params.rs

```diff
@@ -0,0 +1,204 @@
+use crate::core_crypto::commons::dispersion::*;
+use crate::core_crypto::commons::parameters::*;
+use crate::core_crypto::entities::*;
+use crate::core_crypto::prelude::{CastFrom, CastInto, UnsignedInteger};
+use serde::{Deserialize, Serialize};
+
+pub type ClassicBootstrapKeys<Scalar> = (
+    LweBootstrapKeyOwned<Scalar>,
+    LweSecretKey<Vec<Scalar>>,
+    LweSecretKey<Vec<Scalar>>,
+    FourierLweBootstrapKeyOwned,
+);
+
+pub type MultiBitBootstrapKeys<Scalar> = (
+    LweMultiBitBootstrapKeyOwned<Scalar>,
+    LweSecretKey<Vec<Scalar>>,
+    LweSecretKey<Vec<Scalar>>,
+    FourierLweMultiBitBootstrapKeyOwned,
+);
+
+// Fourier key is generated afterward in order to use generic test function
+pub type FftBootstrapKeys<Scalar> = (
+    LweBootstrapKeyOwned<Scalar>,
+    LweSecretKey<Vec<Scalar>>,
+    LweSecretKey<Vec<Scalar>>,
+);
+
+pub type FftWopPbsKeys<Scalar> = (
+    FourierLweBootstrapKeyOwned,
+    LweSecretKey<Vec<Scalar>>,
+    LweSecretKey<Vec<Scalar>>,
+    LwePrivateFunctionalPackingKeyswitchKeyListOwned<Scalar>,
+);
+
+// Fourier key is generated afterward in order to use generic test function
+pub type PackingKeySwitchKeys<Scalar> = (
+    LwePackingKeyswitchKeyOwned<Scalar>,
+    LweSecretKey<Vec<Scalar>>,
+    GlweSecretKey<Vec<Scalar>>,
+);
+
+#[derive(Copy, Clone, Debug, PartialEq, Serialize, Deserialize)]
+pub struct ClassicTestParams<Scalar: UnsignedInteger> {
+    pub lwe_dimension: LweDimension,
+    pub glwe_dimension: GlweDimension,
+    pub polynomial_size: PolynomialSize,
+    pub lwe_modular_std_dev: StandardDev,
+    pub glwe_modular_std_dev: StandardDev,
+    pub pbs_base_log: DecompositionBaseLog,
+    pub pbs_level: DecompositionLevelCount,
+    pub ks_base_log: DecompositionBaseLog,
+    pub ks_level: DecompositionLevelCount,
+    pub pfks_level: DecompositionLevelCount,
+    pub pfks_base_log: DecompositionBaseLog,
+    pub pfks_modular_std_dev: StandardDev,
+    pub cbs_level: DecompositionLevelCount,
+    pub cbs_base_log: DecompositionBaseLog,
+    pub message_modulus_log: CiphertextModulusLog,
+    pub ciphertext_modulus: CiphertextModulus<Scalar>,
+}
+
+#[derive(Clone, Copy, Debug, PartialEq, Serialize, Deserialize)]
+pub struct MultiBitTestParams<Scalar: UnsignedInteger> {
+    pub input_lwe_dimension: LweDimension,
+    pub lwe_modular_std_dev: StandardDev,
+    pub decomp_base_log: DecompositionBaseLog,
+    pub decomp_level_count: DecompositionLevelCount,
+    pub glwe_dimension: GlweDimension,
+    pub polynomial_size: PolynomialSize,
+    pub glwe_modular_std_dev: StandardDev,
+    pub message_modulus_log: CiphertextModulusLog,
+    pub ciphertext_modulus: CiphertextModulus<Scalar>,
+    pub grouping_factor: LweBskGroupingFactor,
+    pub thread_count: ThreadCount,
+}
+
+// Toy parameters to test FFT implementation
```

Comment:
just parameters, what makes them "toys" are their actual values if they are very small, same for all param struct definitions